CHANGELOG.yml

```
@@ -6,6 +6,7 @@ upcoming:
     - Allow for multiple artwork images in My Collection - adam
     - Set price and marketing text dynamically - adam
     - Migrate floor selling price and online asking price in My Bids to the new type - starsirius
+    - remove new inquiry flow feature flag in favor of echo flag - lily
```

One of those small nuances where this is technically what you did, but from the perspective of someone consuming the changelog it may be more helpful to describe what it means. 

```suggestion
    - Enable new fist inquiry flow - lily
```